proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -0,0 +1,103 @@
+
+## CPC Champion (optional)
+
+If you have a champion from the CPC please identify them. If you do not currently have one we will identify someone to support your project.
+
+## Description
+
+A rough description of the project
+
+## Statement of alignment with OpenJS Foundation charter and mission
+
+Please refer to [TBD][]
+
+## Current Code of Conduct
+
+Include a link
+
+## Current Project License
+
+Include a link
+
+## Current Source Control
+
+Link to Github or other source control
+
+## Issues Tracker
+
+Link to GitHub or other tracker
+
+## External Dependencies
+
+Full list of dependencies in your source tree including licenses.
+
+## Release Methodology and mechanics
+
+How often do you release and where are the releases made. Do you have an LTS program?
+
+## List of Committers
+
+Full list of individuals with a commit bit to your project and their corporate affiliation if applicable.
+
+## Governance Structure
+
+* Is there a leadership team?
+* Who are the members of the leadership team?
+* How are members of the leadership team nominated
+* How are commiters nominated?
+
+## Desired Initial Project Phase
+
+At Large / Growth / Impact
+
+## Official Communication Channels
+
+List current channels e.g.: Slack / IRC / Mailing lists
+
+## Project Website
+
+Include link
+
+## Social Media Accounts
+
+Links to social media accounts
+
+## Existing Financial Sponsorship
+
+Does your project currently receive funds? Who do they come from and what are the funds used for?
+
+## Infrastructure Needs or Requests
+
+What needs will your project have from the foundation?
+
+## Onboarding Checklist
+
+This is an informational checklist to help projects onboard into the OpenJS Foundation - tasks we will complete together after your project has been accepted into the foundation. If you have any questions or need help, the OpenJS Foundation CPC is available to assist.
+
+- [ ] Adopt the OpenJS Foundation Code of Conduct
+- [ ] Update project CoC reporting methods to include OpenJS Foundation escalation path
+- [ ] Transfer official domains to OpenJS Foundation
+- [ ] Identify and document other core project infrastructure
+- [ ] If choosing to use a CLA or DCO, make selection and implement appropriate tool
+- [ ] Add or Update Governance.md document (required for Impact stage)
+- [ ] Confirm required files in place (CODE_OF_CONDUCT.md, LICENSE.md)
+- [ ] Project Charter is published on website or github
+- [ ] Update legal copyright notice on project website and github
+- [ ] Add OpenJS Foundation logo to project website
+- [ ] Add Project logo to OpenJS Foundation website; update PROJECTS.md file
+- [ ] Transfer logomark to the OpenJS Foundation
+- [ ] If project is using crowdfunding platforms, add disclaimer to platforms
+- [ ] Identify individuals from the project to join the CPC
+- [ ] Document project and foundation contacts for:
+  * marketing & social media
+  * infrastructure
+  * legal/governance help
```

Comment:
Does Legal need to do some provenance check or something?

Reply:
can you move this question to https://github.com/openjs-foundation/bootstrap/pull/158